modules/update-service-mirror-release.adoc

```diff
@@ -137,31 +137,21 @@ $ oc adm release mirror -a ${LOCAL_SECRET_JSON}  \
 ----
 $ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
 ----
-... Export a local temporary repository name:
+... Take the media to the restricted network environment and upload the images to the local container registry.
```

Comment:
Should end in a colon

Reply:
Rebased onto `main` and fixed the `.` -> `:` with 5fa060baa -> d49d9d62b.